src/Sinch/Fax/Faxes/Utils.cs

```diff
@@ -0,0 +1,431 @@
+using System;
+using System.Collections.Generic;
+
+namespace Sinch.Fax.Faxes
+{
+    internal class FileExtensionContentTypeProvider
+    {
+        private IDictionary<string, string> Mappings { get; set; }
+
+
+        internal FileExtensionContentTypeProvider()
```

Comment:
Where does this list come from? I think it can be trimmed down to the file types supported by the API